wikipedia_revisions/download.py

```diff
@@ -349,15 +349,29 @@ def download_and_parse_files() -> Iterable[Callable[..., Generator[Dict, None, N
             yield lambda: parse_one_file(filename)
 
 
-def _write_rows_to_file(
+def _write_rows_to_pipe(
+    revision_maker: Callable[..., Iterable[Dict]], fields: List[str]
+):
+    pipe_name = f"revisions-{os.getpid()}-{str(time.time()).replace('.', '')}.pipe"
```

Comment:
low prio: naming the pipe based on the name of the input file would be clearer that PID + time

Reply:
not doing this right now since I don't want to add another param, may come back to it later